src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
 // write code here
+const newList = document.querySelector('ul');
```

Comment:
```suggestion
const newList = [...document.querySelectorAll('li')];
```
Тогда не надо будет искать list.children